### DISCUSSION TITLE
Disable Login Button

### DISCUSSION THREAD

**Discussion by ivan200** [rejetto/hfs#639] [Q&A] [answered]
On my server, I only use public access and manage it from the local host. I don't need to create any accounts to differentiate between users with different rights. Is it possible to completely hide the login button from the server so that it doesn't appear in the interface?

**Comment by rejetto** [comment, marked as answer]
no problem, admin-panel > custom > style
```css
#login-button { display:none }
```

if your hfs doesn't have the style section yet, use "html head" and wrap the line above with <style>
```html
<style>
#login-button { display:none }
</style>
```

**Reply by ivan200** [reply]
Thank you, it's working
